docs/admin/resourcequota/walkthrough.md

```diff
@@ -1,164 +1,358 @@
-Note that `kubectl describe rs` works only on kubernetes cluster >= v1.2. If you are running older versions, use `kubectl describe rc` instead.
-If you want to obtain the old behavior, use `--generator=run/v1` to create replication controllers. See [`kubectl run`](/docs/user-guide/kubectl/kubectl_run/) for more details. 
+What happened?  I have no pods!  Let's describe the deployment to get a view of what is happening.
 
 ```shell
-$ kubectl describe rs -l run=nginx --namespace=quota-example
-Name:		nginx-2040093540
-Namespace:	quota-example
-Image(s):	nginx
-Selector:	pod-template-hash=2040093540,run=nginx
-Labels:		pod-template-hash=2040093540,run=nginx
-Replicas:	0 current / 1 desired
-Pods Status:	0 Running / 0 Waiting / 0 Succeeded / 0 Failed
+$ kubectl describe deployment nginx --namespace=quota-example
+Name:                   nginx
+Namespace:              quota-example
+CreationTimestamp:      Mon, 06 Jun 2016 16:11:37 -0400
+Labels:                 run=nginx
+Selector:               run=nginx
+Replicas:               0 updated | 1 total | 0 available | 1 unavailable
+StrategyType:           RollingUpdate
+MinReadySeconds:        0
+RollingUpdateStrategy:  1 max unavailable, 1 max surge
+OldReplicaSets:         <none>
+NewReplicaSet:          nginx-3137573019 (0/1 replicas created)
+...
+```
+
+A deployment created a corresponding replica set and attempted to size it to create a single pod.
+
+Let's look at the replica set to get more detail.
+
+```shell
+$ kubectl describe rs nginx-3137573019 --namespace=quota-example
+Name:                   nginx-3137573019
+Namespace:              quota-example
+Image(s):               nginx
+Selector:               pod-template-hash=3137573019,run=nginx
+Labels:                 pod-template-hash=3137573019
+                        run=nginx
+Replicas:               0 current / 1 desired
+Pods Status:            0 Running / 0 Waiting / 0 Succeeded / 0 Failed
 No volumes.
 Events:
-  FirstSeen	LastSeen	Count	From				SubobjectPath	Type		Reason		Message
-  ---------	--------	-----	----				-------------	--------	------		-------
-  48s		26s		4	{replicaset-controller }			Warning		FailedCreate	Error creating: pods "nginx-2040093540-" is forbidden: Failed quota: quota: must specify cpu,memory
+  FirstSeen	LastSeen	Count From                    SubobjectPath Type      Reason        Message
+  ---------	--------  ----- ----                    -------------	--------  ------        -------
+  4m        7s        11    {replicaset-controller }              Warning   FailedCreate  Error creating: pods "nginx-3137573019-" is forbidden: Failed quota: compute-resources: must specify limits.cpu,limits.memory,requests.cpu,requests.memory
 ```
 
-The Kubernetes API server is rejecting the ReplicaSet requests to create a pod because our pods
-do not specify any memory usage *request*.
+The Kubernetes API server is rejecting the replica set requests to create a pod because our pods
```

Comment:
I think this might sound better as "The Kubernetes API server is rejecting the replica set's request to create a pod ..."